internal/service/alarms/internal/alertmanager/alertmanager.yaml.template

```diff
@@ -19,13 +19,6 @@ route:
     matchers:
       # Always firing alert to verify alertmanager is working
       - alertname=Watchdog
-  - receiver: oran_alarm_receiver
-    group_by:
-      # This label is guaranteed to be present in all alerts
-      - managed_cluster
-    matchers:
-      # Match alerts with a non-empty managed_cluster label
-      - managed_cluster=~".+"
 receivers:
   - name: oran_alarm_receiver
     webhook_configs:
```

Comment:
should we delete send_resolved: true and rely only on the fact that resolved alerts are not present?

Reply:
the main purpose is actually send the notification to user. So when AM calls us that means its time to alert the user  if alarm_sequence_number updates (in our case it would update when resolved or alarm_changed_time)